packages/rspack-test-tools/src/helper/util/checkSourceMap.js

```diff
@@ -1,6 +1,11 @@
 // @ts-nocheck
 // Check the mapping of various key locations back to the original source
-export default async function checkSourceMap(out, outCodeMap, toSearch) {
+export default async function checkSourceMap(
+	out,
+	outCodeMap,
+	toSearch,
+	_checkColumn = true // passing `false` indicating it's an `OriginalSource`.
```

Comment:
The original `checkSourceMap` by default treat every source map passed in containing information for each column. After `run_before_pass` being removed, some of the code only contain line information.